package.json

```diff
@@ -2,14 +2,14 @@
     "name": "qore-vscode",
     "displayName": "Qore",
     "description": "Qore language support for VS Code",
-    "version": "0.3.3",
+    "version": "0.3.4",
```

Comment:
don't we want to increase more then just a patch version? we are forcing user to upgrade qore, I would go with at least minor, maybe even call it version `1.0.0`?

Reply:
That's right. The version should be updated to at least 0.4, but maybe in this case 1.0.0 is a good idea. Also, please add a note about this to `CHANGELOG.md` file, in the same style as the other entries.

Reply:
Actually just noticed that there is already an entry in the changelog. But the version should be increased to 0.4.